spec/routing/trips_routing_spec.rb

```diff
@@ -1,37 +1,39 @@
-require 'rails_helper'
+require "rails_helper"
```

Comment:
Du hast hier alle Single Quotes durch Double Quotes ersetzt - du brauchst doch aber nicht in jedem String Interpolation?